triton-server/src/notification.ts

```diff
@@ -69,18 +70,35 @@ export const sendNotificationEmail = async (
                         -   Readset blocked status count: ${
                             dataset.blocked_status_count
```

Comment:
This should not be sent to clients ...

Reply:
you're too late ;(
Plus don't users deserve to know how many have been blocked?